src/custom-types.json

```diff
@@ -295,6 +295,13 @@
       "http://php.net/manual/en/faq.installation.php"
     ]
   },
+  "application/x-ipynb+json": {
+    "compressible": true,
+    "extensions": ["ipynb"],
```

Comment:
Thank you for your contribution! I checked the reference below, and didn't find anything about this being the file extension for the `application/x-ipynb+json` mime type. But I could be missing something, of course. If you need to add another reference for this, please do so. If it is already on that page, but I am just missing it, please highlight where this information is location on the source linked below. Thank you!

Reply:
Switched to a better source, and also noting that this would close the previously discussed #176

Reply:
Sweet, thank you! Please forgive my misunderstanding still, but the issue comment you linked to says:

> x- or vnd are fine - they're both valid for these "custom" (non-IANA declared). I primarily care about ensuring consistency and documented specs. 😄
> 
> application/x-ipynb+json was declared sometime ago, we can keep on with it.

I guess I'm missing where in that comment it talks about the file extension??

Reply:
Oh, I see! sorry for not understanding the comment about the file extension.

Well, it's actually surprisingly tricky to find a reference for the file extension being .ipynb, but maybe [this link](https://docs.jupyter.org/en/latest/running.html#:~:text=The%20following%20code%20should%20open%20the%20given%20notebook%20in%20the%20currently%20running%20notebook%20server%2C%20starting%20one%20if%20necessary.)?

Reply:
Hmm. The issue is that we have a strong rule on needing a specific reference that clearly provide the information like where it is clear that the specific file extension == that specific mime type. I'm not familiar with Jupyer and there is just not enough context there to link the two together definitively. It has been too frequent that we would accept data only for folks to come argue with us it was wrong or should be changed, over and over. This module is not intended to be a registry, that is what IANA is; it's main purpose is to aggregate the three sources listed at the top of the README in to a JSON file for Node.js users like the Express.js project (the spawn of it).

We of course still accept custom entries, but need good sources from first parties regarding the data before landing so we're not here in a situation of being a registry. I hope that makes sense.

Reply:
Got it. 

If I ask the Jupyter maintainers if they'd state somewhere in their doc the following language:

> Jupyter notebooks use file extension `.ipynb` and their intended mime-type is `application/x-ipynb+json`

Would that work?

Reply:
Absolutely 💯 even just as a comment on their Github would be enough. We have a few like that currently.